source/index.html.md

```diff
@@ -515,8 +515,8 @@ To simulate a transaction failure, create a Payment or Payment Request with an a
 
 For example:
 
-* DE amount `$1.50` will cause the credit transaction to fail, triggering the credit failure code `E150` (Voided By Admin).
-* DE amount `$2.06` will cause the debit transaction to fail, triggering the debit failure code `E206` (Account Not Found).
+* DE amount `$1.50` will cause the credit transaction to fail, triggering the credit failure code `E554-150` (Voided By Admin).
+* DE amount `$2.06` will cause the debit transaction to fail, triggering the debit failure code `E554-206` (Account Not Found).
```

Comment:
Since direct entry is the only payment rail, perhaps we can drop it here (and other places?), as we did with the headings. 

```suggestion
* Amount `$1.50` will cause the credit transaction to fail, triggering the credit failure code `E554-150` (Voided By Admin).
* Amount `$2.06` will cause the debit transaction to fail, triggering the debit failure code `E554-206` (Account Not Found).
```